data/incipias/incipias.txt

```diff
@@ -0,0 +1,335 @@
+		"vocal"
+
+phrase "vocal"
+	word
+		"a"
+		"e"
+		"i"
+		"o"
+		"u"
+
+phrase "vocal (capitalized)"
+	word
+		"A"
+		"E"
+		"I"
+		"O"
+		"U"
+
+phrase "consonant"
+	word
+		"b"
+		"d"
+		"g"
+		"k"
+		"l"
+		"m"
+		"n"
+		"r"
+		"s"
+		"t"
+
+phrase "consonant (capitalized)"
+	word
+		"B"
+		"D"
+		"G"
+		"K"
+		"L"
+		"M"
+		"N"
+		"R"
+		"S"
+		"T"
+
+phrase "hex color"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+	phrase
+		"hex digit"
+
+phrase "hex digit"
+	word
+		"0"
+		"1"
+		"2"
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+		"a"
+		"b"
+		"c"
+		"d"
+		"e"
+		"f"
+
+fleet "Hicemus Standard Surveilance"
+	government "Hicemus"
+	names "hicemus military"
+	cargo 0
+	personality
+		heroic vindictive
+	variant 10
+		"Nimbus"
+	variant 4
+		"Nimbus" 2
+	variant 4
+		"Venta"
+
+fleet "Incipias Standard Mining"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 0
+	personality
+		timid mining harvests coward
+	variant 2
+		"Nimbus (Mining)"
+	variant 5
+		"Nimbo Stratus"
+	variant 5
+		"Nimbo Stratus" 2
+	variant 1
+		"Nimbo Stratus"
+		"Nimbus (Mining)"
+
+fleet "Incipias Standard Trade"
+	government "Incipias Civilian"
+	names "incipias civilian"
+	cargo 50
+	personality
+		confusion 40
+		timid appeasing
+	variant 5
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbus (Civil)"
+	variant 4
+		"Nimbo Cirrus" 2
+	variant 3
+		"Gero"
+	variant 2
+		"Gero" 2
+	variant 2
+		"Gero"
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbo Cirrus" 2
+		"Nimbus (Civil)"
+
+outfitter "Hicemus"
+	"Plasma Discharger"
+	"Stagnation Beam"
+	"Gas-Class Shield"
+	"Liquid-Class Shield"
+	"Metallic Hydrogen Cell"
+	"Cloud Piercer"
+
+	"MH Blue-Class Engine"
+	"MH Blue-Class Steering"
+	"MH Green-Class Engine"
+	"MH Green-Class Steering"
+	"MH Red-Class Engine"
+	"MH Red-Class Steering"
+
+shipyard "Hicemus"
+	"Nimbo Cirrus"
+	"Nimbo Stratus"
+	"Nimbus"
+	"Gero"
+
+news "Incipias common"
+	location
+		government "Conlatio" "Hicemus"
+	name
+		word
+			"Incipias Captain"
+			"Incipias Crew Member"
+			""
+	message
+		phrase
+			"incipias common hard defined"
+			"incipias common praise"
+
+phrase "incipias common"
+	phrase
+		"incipias common hard defined"
+		"incipias common praise"
+
+phrase "incipias common hard defined"
+	word
+		"You say you left this [system, home]? One day, [I, we] will also see other [systems, homes]."
+		"The past was a more [interesting, dangerous] time to fly a spaceship. Nowadays, a lot of people have one."
+		"I just got my license for being part of a spaceship renewed. The government controls them [strictly, like freaks]."
+		"There is nothing more [exciting, fun] than to switch to turbine mode and fly through the clouds of [our home, our origin]."
+		"It has been [some, a lot of] time since the last great breakthrough has been achieved. The government [says, promises us] that they are working on something, but my guess is that they have no clue what to do."
```

Comment:
What kind of breakthrough are they talking about here? Technological? Political? Scientific?

Reply:
technological